cumulusci/cumulusci.yml

```diff
@@ -125,6 +125,10 @@ tasks:
         description: Creates a Community in the target org using the Connect API
         class_path: cumulusci.tasks.salesforce.CreateCommunity
         group: Salesforce Communities
+    create_network_members:
```

Comment:
Overall this looks great and almost ready to merge, but I find it a bit confusing that the task config is named `create_network_members` since it creates NetworkMemberGroups, not NetworkMembers. Should it be named `create_network_member_groups` for clarity?

Reply:
That is a great point - I will make the change